src/main/java/fi/metatavu/metamind/server/rest/SystemApi.java

```diff
@@ -89,30 +86,30 @@ public Response reload() {
     return Response.ok().build();
   }
 
-  @GET
-  @Path ("/slotModel/{name}")
-  @Produces({ "application/octet-stream" })
-  public Response getSlotModelData(@PathParam ("name") String name) {
-    SlotModel slotModel = modelsContoller.findSlotModelByName(name);
-    if (slotModel == null) {
-      return respondNotFound();
-    }
-    
-    return streamResponse(slotModel.getData(), "application/octet-stream");
-  }
-  
-  @GET
-  @Path ("/intentModel/{name}")
-  @Produces({ "application/octet-stream" })
-  public Response getIntentModelData(@PathParam ("name") String name) {
-    IntentModel intentModel = modelsContoller.findIntentModelByName(name);
-    if (intentModel == null) {
-      return respondNotFound();
-    }
-    
-    return streamResponse(intentModel.getData(), "application/octet-stream");
-  }
-  
+//  @GET
+//  @Path ("/slotModel/{name}")
+//  @Produces({ "application/octet-stream" })
+//  public Response getSlotModelData(@PathParam ("name") String name) {
+//    SlotModel slotModel = modelsContoller.findSlotModelByName(name);
+//    if (slotModel == null) {
+//      return respondNotFound();
+//    }
+//    
+//    return streamResponse(slotModel.getData(), "application/octet-stream");
+//  }
+//  
+//  @GET
+//  @Path ("/intentModel/{name}")
+//  @Produces({ "application/octet-stream" })
+//  public Response getIntentModelData(@PathParam ("name") String name) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)